mautic/mautic
# [UX] Change "When a contact..." to improve accessibility

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
Imagine that you are not seeing all the content on the screen and only one reader speaks a label for the entire page while you fill in the fields. When the screen reader reaches this label, the way it is written does not provide enough context and may sound random.

This update enhances accessibility by revising the ambiguous “When a contact…” label to the more descriptive “Action taken by user”. This change provides better context for screen reader users, ensuring that each label is self-explanatory and contributes to a coherent understanding of the page content. By improving label clarity, we facilitate a more inclusive and navigable experience for all users.



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Point > Manage actions
3. Look at the new label

![image](https://github.com/mautic/mautic/assets/116097999/ac28e3d0-e4c0-4a3b-a05d-ffbf22254a45)
